examples/jenkins/application-template.json

```diff
@@ -314,7 +349,27 @@
     }
   ],
   "parameters": [
+     {
+      "name": "DEPLOYMENT_REQUEST_SIZE",
+      "description": "Memory rquested on startup from OpenShift Quota System",
+      "value": "64Mi"
+    },
     {
+      "name": "DEPLOYMENT_LIMIT_SIZE",
+      "description": "Maximum amount of memory allowed by OpenShift Quota System",
+      "value": "128Mi"
+    },
+     {
+      "name": "BUILD_REQUEST_SIZE",
+      "description": "Memory rquested on startup from OpenShift Quota System",
+      "value": "32Mi"
+    },
+    {
+      "name": "BUILD_LIMIT_SIZE",
```

Comment:
I see ... nuking the build request then.

And these change (if we keep them at all) should presumably only go in then after the project quota defaults changes go in.